docs/staging-environments/prosody-std/prosody.cfg.lua

```diff
@@ -0,0 +1,244 @@
+-- For the "sql" backend, you can uncomment *one* of the below to configure:
+--sql = { driver = "SQLite3", database = "prosody.sqlite" } -- Default. 'database' is the filename.
+--sql = { driver = "MySQL", database = "prosody", username = "prosody", password = "secret", host = "localhost" }
+--sql = { driver = "PostgreSQL", database = "prosody", username = "prosody", password = "secret", host = "localhost" }
+
+
+-- Archiving configuration
+-- If mod_mam is enabled, Prosody will store a copy of every message. This
+-- is used to synchronize conversations between multiple clients, even if
+-- they are offline. This setting controls how long Prosody will keep
+-- messages in the archive before removing them.
+
+archive_expires_after = "1w" -- Remove archived messages after 1 week
+
+-- You can also configure messages to be stored in-memory only. For more
+-- archiving options, see https://prosody.im/doc/modules/mod_mam
+
+-- Logging configuration
+-- For advanced logging see https://prosody.im/doc/logging
+log = {
+    {levels = {min = "debug"}, to = "console"};
+}
+
+-- Uncomment to enable statistics
+-- For more info see https://prosody.im/doc/statistics
+-- statistics = "internal"
+
+-- Certificates
+-- Every virtual host and component needs a certificate so that clients and
+-- servers can securely verify its identity. Prosody will automatically load
+-- certificates/keys from the directory specified here.
+-- For more information, including how to use 'prosodyctl' to auto-import certificates
+-- (from e.g. Let's Encrypt) see https://prosody.im/doc/certificates
+
+-- Location of directory to find certificates in (relative to main config file):
+certificates = "certs"
+
+-- HTTPS currently only supports a single certificate, specify it here:
+--https_certificate = "/etc/prosody/certs/localhost.crt"
+
+-- custom config for http_upload module
+http_upload_file_size_limit = 10485760 -- 10 Mb in bytes
+http_upload_expire_after = 60 * 60 * 24 * 1 -- a day in seconds
+http_upload_quota = 209715200 -- 200 Mb in bytes
+
+----------- Virtual hosts -----------
+-- You need to add a VirtualHost entry for each domain you wish Prosody to serve.
+-- Settings under each VirtualHost entry apply *only* to that host.
+consider_websocket_secure = true
+cross_domain_websocket = { "http://localhost", "https://localhost", "http://localhost:3000" }
+
+disco_items = {
+	{ "proxy.localhost", "SOCKS5 file transfert proxy" };
+	{ "conference.localhost", "Multi users chat" };
+	{ "upload.localhost", "File upload" };
+}
+
+VirtualHost "localhost"
+	name = "XMPP server"
+	enabled = true
+
+Component "proxy.localhost" "proxy65"
+	name = "SOCKS5 file transfert proxy service"
+	proxy65_address = "localhost"
+	proxy65_acl = { "localhost" }
+
+Component "conference.localhost" "muc"
+	modules_enabled = {
+			"muc_mam";
+			"vcard_muc";
+	}
+	name = "Conferences server"
+	restrict_room_creation = "local"
+	max_history_messages = 50
+
+-- Component "upload.localhost" "http_upload"
```

Comment:
> The additional component for proxy65 is redundant (and the one for http_upload was commented out also)

Yes, I do not remember if it was in the default .cfg file. I'll remove it.

Reply:
Removal of `http_upload` from `modules_enabled` and add it as a component according to the [module documentation](https://modules.prosody.im/mod_http_upload)

Reply:
Yeah it actually works, it's just that some clients don't handle this case so it's annoying for support and they decided to write the documentation this way.

Reply:
Should I put back the global module loaded?

Reply:
No it's ok either way. It would be cool though if both cases were supported :)

Reply:
Both was working for me with XMPP web 😆